<launch>
  <node name="rtk_driver" pkg="gps_driver" type="rtk_driver.py" output="screen">
    <param name="port" type="string" value="$(arg port)"/>
    <param name="baudrate" type="int" value="4800"/>
    <param name="sampling_rate" type="double" value="5.0"/>
  </node>
</launch>

